react/features/mobile/callkit/middleware.js

```diff
@@ -199,16 +212,34 @@ function _conferenceJoined(store, next, action) {
  * @private
  * @returns {*} The value returned by {@code next(action)}.
  */
-function _conferenceLeft(store, next, action) {
-    const result = next(action);
-
-    const { callUUID } = action.conference;
+function _sessionEnded(store, next, action) {
+    const callUUID = _getCallUUIDForSessionAction(store, action);
 
     if (callUUID) {
         CallKit.endCall(callUUID);
     }
 
-    return result;
+    return next(action);
+}
+
+/**
+ * FIXME.
+ *
+ * @param {Store} store - FIXME.
+ * @param {Object} action - FIXME.
+ * @returns {string|undefined}
+ * @private
+ */
+function _getCallUUIDForSessionAction(store, action) {
+    const url = action.session.url;
+    const session = getSession(store, url);
+    const callUUID = session && session.callkit && session.callkit.callUUID;
```

Comment:
As I mentioned before I think it would be a good idea to make the UUID a more integral part of the whole thing.